frameworks/python/django/src/testApp/test/test_model_crud/test_crud.py

```diff
@@ -0,0 +1,46 @@
+# These are testse
+
+from django.test import TestCase
+from testApp.models import Person
+
+class CrudTestCase(TestCase):
```

Comment:
Here could we also include the link to the guide that is tested as a comment.

Reply:
I was planning on making a readme file for this directory explaining what is tested and how it is tested